flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/DefaultScheduler.java

```diff
@@ -245,23 +260,23 @@ public void allocateSlotsAndDeploy(final Collection<ExecutionVertexDeploymentOpt
 		}
 	}
 
+	private void validDeploymentOptions(final Collection<ExecutionVertexDeploymentOption> deploymentOptions) {
```

Comment:
How about `validateDeploymentOptions()`?

Reply:
You are right. I left some letters behind.

Reply:
Done.